apps/engine/.justfile

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env just --justfile
+
+set fallback
+
+repo := `git rev-parse --show-toplevel`
+profile := env_var_or_default('PROFILE', "dev")
+
+[private]
+@default:
+  just usage
+
+# Copied from `/.justfile` to ignore the `lint-toml` job.
```

Comment:
Any alternatives you can think of?

Reply:
Agreed, but I more meant isn't there a better way to override this without copying it? Can we not locally redefine `lint-toml` to be a no-op like we do for `miri` elsewhere for example?

Reply:
Good idea: [Ignore lint-toml job in engine](https://github.com/hashintel/hash/pull/2292/commits/9ed789eeadb147c028298f2d45f099c304dc2ace)